packages/neutron-sdk/src/bindings/marketmap/query.rs

```diff
@@ -0,0 +1,38 @@
+use crate::bindings::marketmap::types::{MarketMap, Params};
+use schemars::JsonSchema;
+use serde::{Deserialize, Serialize};
+
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, Eq, JsonSchema)]
+#[serde(rename_all = "snake_case")]
+pub enum MarketmapQuery {
+    /// Parameters queries the parameters of the module.
+    Params {},
+    LastUpdated {},
+    MarketMap {},
+}
+
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, Eq, JsonSchema)]
+#[serde(rename_all = "snake_case")]
+pub struct ParamsResponse {
+    pub params: Params,
+}
+
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, Eq, JsonSchema)]
+#[serde(rename_all = "snake_case")]
+pub struct LastUpdatedResponse {
+    pub last_updated: u64,
+}
+
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, Eq, JsonSchema)]
+#[serde(rename_all = "snake_case")]
+pub struct MarketMapResponse {
+    // MarketMap defines the global set of market configurations for all providers
+    // and markets.
+    pub market_map: MarketMap,
+    // LastUpdated is the last block height that the market map was updated.
```

Comment:
can we do all field comments in the typical rust docs manner?
like `/// **owner_address** is an address of contract which registered interchain account`